docker-entrypoint.sh

```diff
@@ -6,4 +6,4 @@ echo "==> $(date +%H:%M:%S) ==> Migrating Django models..."
 python src/manage.py migrate --noinput
 
 echo "==> $(date +%H:%M:%S) ==> Running Gunicorn..."
-exec gunicorn -c /app/src/config/gunicorn.py config.wsgi -b 0.0.0.0:${GUNICORN_BIND_PORT} --chdir /app/src/
+exec gunicorn -c /app/src/config/gunicorn.py config.wsgi -b ${GUNICORN_BIND_SOCKET} --chdir /app/src/
```

Comment:
Do not remove the http entry, you can add both. It's good to have that so we can configure Kubernetes to check the pod without nginx